File: Starter Problems 2/dish.py
op = open('dishin.txt', 'r')
n = []
c = int(op.readline().strip())
for c in range(0, c):
	n.append(int(op.readline()))

ma = max(n)
mn = min(n)
to = 0
for i in n:
	to += i
to = to/len(n)

open('dishout.txt', 'w').write(str(mn) + ' ' + str(ma) + ' ' + str(to))

File: Starter Problems 2/dish.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ifstream inFile;
    inFile.open("dishin.txt");
    int n;
    inFile >> n;
    int nums[n];
    for (int i = 0;i<n;i++) {
        inFile >> nums[i];
    }
    inFile.close();
    int minimum,maximum,mean;
    minimum = *min_element(nums,nums+n);
    maximum = *max_element(nums,nums+n);
    int total = 0;
    for (int i = 0;i<n;i++) {
        total += nums[i];
    }
    mean = total/n;
    ofstream outFile;
    outFile.open("dishout.txt");
    outFile << minimum << " " << maximum << " " << mean;
    outFile.close();
}
